apps/builder/app/builder/features/navigator/navigator-tree.tsx

```diff
@@ -608,14 +628,28 @@ export const NavigatorTree = () => {
         {flatTree.map((item) => {
           const level = item.visibleAncestors.length - 1;
           const key = item.selector.join();
-          const propValues = propValuesByInstanceSelector.get(
+          const propValues = propValuesByInstanceSelectorWithMemoryProps.get(
             getInstanceKey(item.selector)
           );
           const show = Boolean(propValues?.get(showAttribute) ?? true);
+
+          // Hook memory prop
+          const isAnimationSelected =
+            propValues?.get(animationCanPlayOnCanvasAttribute) === true;
+
+          const isAnimationPinned =
+            item.instance.component ===
+              "@webstudio-is/sdk-components-animation:AnimateChildren" &&
+            (propValues?.get("action") as AnimationAction)?.isPinned === true;
```

Comment:
huge lock on library, can you check prop type instead?

Reply:
Done